### PR TITLE
Adding patches for hal layer

### PR DESCRIPTION
**Pull request by anatar818**
Adding patches and updating the bbappend files

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi Aishwarya: Patches should have headers.  At a minimum, please add a header to your patch files to indicate the source of the patch: Source: Comcast, or RDK or an email or an external URL etc.  Thanks!

**Comment by anatar818** [comment]
Done @mhughesacn . thanks

**Comment by mhughesacn** [comment]
Many thanks.

**Review by cpokuru — COMMENTED** [review]
We need to upstream to hal repo

File: meta-rdk-mtk-bpir4/recipes-ccsp/hal/files/Add_interface_Changes.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Index: ethsw/rdkb_hal/src/ethsw/ccsp_hal_ethsw.c
+===================================================================
+--- ethsw.orig/rdkb_hal/src/ethsw/ccsp_hal_ethsw.c
++++ ethsw/rdkb_hal/src/ethsw/ccsp_hal_ethsw.c
+@@ -40,7 +40,7 @@
+ #define LM_ARP_ENTRY_FORMAT  "%63s %63s %63s %63s %17s %63s"
+ 
+ #define ETH_WAN_INTERFACE  "erouter0"
+-#define ETH_WAN_IFNAME   "eth2"
++#define ETH_WAN_IFNAME   "lan0"
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-ccsp/hal/files/Add_interface_Changes.patch:13]: This need to be under flag

**Review by cpokuru — APPROVED** [review]
once mediatek upstream changes we will revert this . main branch should not contain any patches

**Pull request merged**